preinstall.js

```diff
@@ -3,3 +3,5 @@
 if (process.env.npm_execpath.indexOf('npm') === -1) {
     throw new Error('\x1b[31mYou must use "npm install", yarn is not supported\x1b[0m');
 }
+
+require('./symlink.js');
```

Comment:
think we should only require this if the script is executed inside of the `assets/admin` folder